jwala-tomcat/src/main/resources/data/scripts/linux/jvm-service.sh

```diff
@@ -12,115 +12,27 @@
 ### END INIT INFO
 
 # Source LSB function library.
-. /etc/init.d/functions
 
-NAME=`basename "$0"`
+[Unit]
+Description= jwala service file
+After=network.target
 
-TOMCAT_HOME=@TOMCAT_HOME@
-TOMCAT_HOME_BIN=$TOMCAT_HOME/bin
+[Service]
+Type=forking
 
 # Define the tomcat username
-TOMCAT_USER="${TOMCAT_USER:-tomcat}"
+User=tomcat
```

Comment:
Add the Group option as well, and the value should be a parameter (TOMCAT_GROUP)